tox.ini

```diff
@@ -20,6 +20,7 @@ extras =
     coverage
 deps =
   django32: Django~=3.2.0
+  pyquery==2.0.0
```

Comment:
please specify this dependency in `setup.cfg` like with the other test dependencies